src/Ingredients/IngredientGateway.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+
+namespace BreadAndIfit\Ingredients;
+
+
+use BreadAndIfit\Interfaces\GatewayInterface;
+class IngredientGateway implements GatewayInterface
+{
+    /**
+     * Gets data from API
+     *
+     * @param array $ingredients The list of user selected ingredients
+     * @return string JSON of recipes
+     */
+    static public function sendDataReturnResponse(array $ingredients): string
+    {
+        $recipes = self::sendDataToAPI(self::formatData($ingredients));
+        return $recipes;
+    }
+
+    /**
+     * Formats ingredients array into url for http request
+     *
+     * @param array $ingredients The list of user selected ingredients
+     * @return string returns url for API filled with query data
+     */
+    static private function formatData(array $ingredients): string
+    {
+        $url = "http://www.recipepuppy.com/api/?i=";
+        foreach ($ingredients as $ingredient) {
+            $url .= $ingredient . ',';
+        }
+        $url = substr($url, 0, -1);
+        return $url;
```

Comment:
should use https://www.php.net/manual/en/function.urlencode.php on the url before returning it to make it a safe url

Reply:
done